cmake/libenvpp.cmake

```diff
@@ -0,0 +1,14 @@
+include_directories(${CMAKE_SOURCE_DIR}/3rdparty/libenvpp/include)
+include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/3rdparty/libenvpp/external/fmt/include)
```

Comment:
Please, avoid using 3p deps of other 3p deps

Reply:
It's header has problem with clang tidy, please help with other solution

Reply:
I meant not to remove this `include_directories` statement, but rather avoid using things from there (e.g. format output). Especially, considering the fact that C++20 has built-in `std::format`. I suggest sticking with standard things at this point

Reply:
> fmt is part of libenv library and without it library can't build

No problem here